src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -360,6 +368,12 @@ export const WorkspaceMenuTrigger = ({ children, canShare, isOwner, setCloningWo
         tooltipSide: 'left',
         onClick: () => setSharingWorkspace(true)
       }, [makeMenuIcon('share'), 'Share']),
+      h(MenuButton, {
+        disabled: !isOwner,
+        tooltip: !isOwner && 'You have not been granted permission to toggle the lock for this workspace',
+        tooltipSide: 'left',
+        onClick: () => setTogglingWorkspaceLock(true)
+      }, isLocked ? [makeMenuIcon('unlock'), 'Unlock'] : [makeMenuIcon('lock'), 'Lock']),
       h(MenuButton, {
         'aria-label': 'Workspace delete',
         disabled: !isOwner,
```

Comment:
This button should be disabled when the workspace is locked since Rawls now rejects requests to delete a locked workspace.